.github/workflows/prepare-release.yml

```
@@ -0,0 +1,23 @@
+name: Prepare Buildpack Releases
```

(there is just the one buildpack for now)
```suggestion
name: Prepare Buildpack Release
```